braze_api.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.7'
 
-  spec.add_runtime_dependency 'faraday', '~> 0.15'
+  spec.add_runtime_dependency 'faraday', '>= 0.15', '< 1.0.0'
```

Comment:
`< 1.0.0` is more restrictive than your #2 PR request. Will the gem work with
`spec.add_runtime_dependency 'faraday', '>= 0.15'`?